projects/vdk-plugins/vdk-impala/.plugin-ci.yml

```diff
@@ -1,7 +1,7 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-image: "python:3.7"
```

Comment:
what happens when you remove this image? It just uses a default one from gitlab? 
If so what is it? 
I'm just curious, I will still approve.

Reply:
There's a default one specified in the root gitlab-ci.yaml (https://github.com/vmware/versatile-data-kit/blob/main/.gitlab-ci.yml#L4)


There is a potential problem in specifying the same "global" key in multiple yaml files, it gets overwritten.
And that is global key. It is not scoped to anything (unlike per-job image which is scoped to a specific gitlab ci job)

The problem can surface because the override is not scoped to the file. So depending on the import order you may get surprising values of an image in some cases in some files. E.g you are running python tests and all of a sudden